evennia/utils/eveditor.py

```diff
@@ -602,6 +652,41 @@ def func(self):
             # set echoing on/off
             editor._echo_mode = not editor._echo_mode
             caller.msg("Echo mode set to %s" % editor._echo_mode)
+        elif cmd == ":-":
```

Comment:
Since the EvEditor mimcs VIM, the increase indent should be `:>` and decrease indent `:<`. The `:=` is just fine though.